src/components/PaymentLog.vue

```diff
@@ -1142,10 +1231,41 @@ export default {
         this.rPayment = rePayment
         this.pPrice = total
         this.test1 = false
-        const months = this.rDuration / 30
 
+        //gray out button if not verified for NoBS
```

Comment:
Will we still use this? if not delete

Reply:
yes, in case we need to put the verification check again